src/resource/factory/SoH/DisplayListFactory.cpp

```diff
@@ -8,8 +8,8 @@ DisplayList* DisplayListFactory::ReadDisplayList(BinaryReader* reader) {
 
     switch (version) {
         case Version::Deckard: {
-            DisplayListV0 DL = DisplayListV0();
-            DL.ParseFileBinary(reader, dl);
+            DisplayListV0 dlFac = DisplayListV0();
```

Comment:
maybe `dlFactory`? `dlFac` isn't a clear name to me

Reply:
Out of scope.